exonum_launcher/launcher.py

```diff
@@ -52,7 +52,10 @@ def _load_clients(self) -> List[ExonumClient]:
 
     def _load_runtime_plugins(self) -> Dict[str, RuntimeSpecLoader]:
         runtime_loaders: Dict[str, RuntimeSpecLoader] = dict()
-        for runtime_name, class_path in self.config.plugins["runtime"].items():
+        plugins = self.config.plugins
```

Comment:
IMHO it'll be better to handle this in `configuration.py`, so the `Configuration` object will always have the same inner structure which `Launcher` can rely on.

Reply:
Thanks for the tip! Added initialization there.